scripts/sync_world_map_langs.py

```
@@ -0,0 +1,522 @@
+                    "id": s_zh["id"],
+                    "zh": s_zh["name"].strip(),
+                    "en": s["en"], "ja": s["ja"], "ko": s["ko"],
+                })
+            level_list.append({
+                "id": lv_zh["id"],
+                "name": {l: lv[l]["name"] for l in langs},
+                "sites": sites,
+            })
+        levels.append({"id": m_zh["id"], "name": {l: m[l]["name"] for l in langs}, "levels": level_list})
+
+    # catalog: 按 mainType/subType 顺序对齐（四语 subType id 一致）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)